### ISSUE TITLE
Import Owner / User / Author

### ISSUE THREAD

**Issue by KCErb** [larowlan/default_content#57]
Hi there,

When I've used this module to export and then import entities everything stay intact except for user info (sometimes called Owner, User, and Author by various modules). 

The import process seems to pretty consistently fail to handle user referenced correctly. In all cases, I've just used the default admin account to create things before export `uid: 1` . And then on import some modules have no reference and some change the reference to the default anonymous user `uid: 0`.

Is this a known issue?

**Comment by larowlan** [comment]
No, its a new issue - perhaps you could add to the existing test a case that shows the failure as a first step?
